pdfarranger/pdfarranger.py

```diff
@@ -66,14 +67,21 @@
 
 DOMAIN = 'pdfarranger'
 ICON_ID = 'com.github.jeromerobert.' + DOMAIN
+
+def get_libintl_path():
+    f = 'libintl.so.8'
+    if os.name == 'nt':
+        f = 'libintl-8'
+    if platform.system() == 'Darwin':
+        f = 'libintl.8.dylib'
+    return f
```

Comment:
Can we get rid of the variable `f` here and return directly? Reorder the statements if overwriting was part of the logic but I guess that's not necessary here.
Also add an empty line after the function for readability.